docs/search/search_for_content.md

```diff
@@ -3,26 +3,46 @@ description: Search through all content in the system and filter the results by
 ---
 
 # Search for content
+
 You can enter a search keyword and the application checks all the searchable Fields and returns a list of search results that contain the keyword.
 
 ![Basic Search](img/basic_search.png)
 
-You can also select a language to search in.
-The results will contain Content items that are translated into this language.
+Everywhere in the Back Office, completion suggestions are proposed while you type in the search box.
+In addition to clicking in it, you can also move focus to search bar using the keyboard shortcut `Ctrl+/` on Windows/Linux, or `Cmd+/` on macOS.
+
+- If a suggestion is what you're looking for, to use it, click on it in the suggestion drop-down, or use `down` and `up` keys to highlight it, and then press the `Enter` key.
+
+- If no suggestion corresponds to your search, press the `Enter` key or click **View all results**.
+
+![Suggestion drop-down](img/suggestion.png)
 
+On the results page, if several languages are available, you can select a language to search in at the top of the left **Filters** menu.
+The results contain Content items that are translated into this language.
 The result list also shows which languages the Content item is translated to.
 
+By default, the results are sorted by decreasing relevance.
+By selecting an option from a drop-down menu in the top right corner, you can sort them by name, publication date, or modification date.
+
+![Sorting menu](img/sorting_menu.png)
+
 ## Filtered search
 
 Filtered search allows you to filter by:
 
- - Content Type
- - Section
- - Modified date
- - Created date
- - Creator
- - Subtree
+- Language (if more than one language available)
+- Content type
+- Modified date
+- Created date
+- Creator
+- Section
+- Subtree
+
+All new or custom content types, sections and languages are automatically added to the drop-down filter lists.
+
+A Content item count is indicated between parenthesis for content type and section filters (if your search engine allows faceting, unlike the Legacy one).
```

Comment:
Here, instead of parenthesis that says "if your search engine allows faceting, unlike the Legacy one" I'd say something like "Depending on the system configuration, a Content item count can be indicated for content type and section filters."
End users may not know what faceted means and what search engine is used in their installation.

Reply:
You're right, this is use doc.
```suggestion
Depending on the system configuration, a Content item count can be indicated for content type and section filters.
```